### PR TITLE
Adds the song "Sole Survivor" by Blue Öyster Cult.

### PR DESCRIPTION
**Pull request by Xoxeyos**
The link for the song is here, for your enjoyment. https://www.youtube.com/watch?v=nRjLv1L0WF8

🆑
rscadd: Adds the Song "Sole Survivor" to the title screen.
/🆑

### CODE REVIEW

**Comment by boodaliboo** [comment]
Wouldn’t it be funny if council updated he configs for the last song update

**Review by Hopekz — APPROVED** [review]

**Comment by swissloaf** [comment]
> Wouldn’t it be funny if council updated he configs for the last song update

all songs are through the code now, not the config

**Comment by boodaliboo** [comment]
> > Wouldn’t it be funny if council updated he configs for the last song update
> 
> all songs are through the code now, not the config

i was a dumbass somehow my song pr that got merged got reverted

File: yogstation/code/controllers/subsystem/ticker.dm [hunk 1]
```diff
@@ -29,6 +29,7 @@
 		"https://www.youtube.com/watch?v=fMn30T5wEVg",						// Mike Morasky - SynTek Residential Mall (Extended by FoxOnTheRails)
 	    "https://www.youtube.com/watch?v=iB5Az3vbkYs",						// Coalescence - Risk of Rain (Cover by Nahu Pyrope)
 	    "https://www.youtube.com/watch?v=SQOdPQQf2Uo",						// Star Trek The Motion Picture: Main Theme Album Style Edit
+		"https://www.youtube.com/watch?v=nRjLv1L0WF8",                      // Blue Oyster Cult: Sole Survivor
```
Comment (alexkar598) [inline yogstation/code/controllers/subsystem/ticker.dm]:
```suggestion
		"https://www.youtube.com/watch?v=nRjLv1L0WF8",                      // Blue Oyster Cult - Sole Survivor
```

**Review by alexkar598 — CHANGES_REQUESTED** [review]
see above

**Review by alexkar598 — APPROVED** [review]

**Comment by nesisknesis** [comment]
ill leave it in for a week or so

**Pull request merged**